MH4U Dex/MonsterItemTableViewController.m

```diff
@@ -1,45 +1,56 @@
 //
-//  MonsterLowRankTableViewController.m
+//  MonsterItemTableViewController.m
 //  MH4U Dex
 //
 //  Created by Joseph Goldberg on 2/9/15.
 //  Copyright (c) 2015 Joseph Goldberg. All rights reserved.
 //
 
-#import "MonsterLowRankTableViewController.h"
+#import "MonsterItemTableViewController.h"
+
+#import <CoreData/CoreData.h>
+
+#import "ItemContainerViewController.h"
 #import "MonsterEncyclopediaViewController.h"
+
+#import "Item.h"
 #import "Monster.h"
 #import "MonsterDrop.h"
-#import <CoreData/CoreData.h>
-#import "Item.h"
+
 #import "DropTableViewCell.h"
-#import "ItemContainerViewController.h"
 
-@interface MonsterLowRankTableViewController ()
+@interface MonsterItemTableViewController ()
 
 @property (nonatomic, strong) NSArray *drops;
 
 @end
 
-@implementation MonsterLowRankTableViewController
+@implementation MonsterItemTableViewController
 
 - (void)viewDidLoad
 {
     [super viewDidLoad];
     NSError *fetchError = nil;
     NSFetchRequest *monsterFetchRequest = [[NSFetchRequest alloc] initWithEntityName:[Monster entityName]];
-    NSPredicate *monsterPredicate = [NSPredicate predicateWithFormat:@"%K == %@", @"name", self.monster];
+    NSPredicate *monsterPredicate = [NSPredicate predicateWithFormat:@"%K == %@", MonsterAttributes.name, self.monster];
     [monsterFetchRequest setPredicate:monsterPredicate];
     Monster *monster = (Monster *)[self.managedObjectContext executeFetchRequest:monsterFetchRequest error:&fetchError].firstObject;
     
     NSFetchRequest *fetchRequest = [[NSFetchRequest alloc] initWithEntityName:[MonsterDrop entityName]];
-    NSSortDescriptor *sortDescriptor = [NSSortDescriptor sortDescriptorWithKey:@"method" ascending:YES];
-    NSPredicate *predicate = [NSPredicate predicateWithFormat:@"%K == %@", MonsterDropRelationships.monsterSource, monster];
-    [fetchRequest setPredicate:predicate];
+    NSSortDescriptor *sortDescriptor = [NSSortDescriptor sortDescriptorWithKey:MonsterDropAttributes.method ascending:YES];
+    NSPredicate *dropPredicate = [NSPredicate predicateWithFormat:@"%K == %@", MonsterDropRelationships.monsterSource, monster];
+    NSPredicate *rankPredicate;
+    //TODO: Clean this up a little bit.
+    if (self.rank) {
+        rankPredicate = [NSPredicate predicateWithFormat:@"%K == %@", MonsterDropAttributes.rank, self.rank];
+    } else {
+        rankPredicate = [NSPredicate predicateWithFormat:@"%K == %@", MonsterDropAttributes.rank, @"Low"];
+    }
```

Comment:
You could rewrite these 5 lines as:
```objc
rankPredicate = [NSPredicate predicateWithFormat:@"%K == %@", MonsterDropAttributes.rank, self.rank ?: @"Low"];
```
The `?:` operator returns the first input if it's not `nil`/false-y, otherwise it returns the second input.

Is there a constant for `@"Low"`?  If not, would it make sense to define one?